src/js/claims-status/components/appeals-v2/Expander.jsx

```diff
@@ -31,10 +31,9 @@ const Expander = ({ expanded, dateRange, onToggle, missingEvents }) => {
 
 Expander.propTypes = {
   expanded: PropTypes.bool.isRequired,
-  missingEvents: PropTypes.bool.isRequired,
   dateRange: PropTypes.string.isRequired,
   onToggle: PropTypes.func.isRequired,
-  cssClass: PropTypes.string.isRequired,
+  missingEvents: PropTypes.bool.isRequired,
```

Comment:
^ FYI missingEvents was already a proptype, but I re-ordered it to match the order in which props are destructured in the function definition. Only thing that's really changing is no more cssClass proptype.